site/docs/components/avatar-group.mdx

```diff
@@ -0,0 +1,95 @@
+---
+title: Avatar Group
+navTitle: Avatar Group
+summaryParagraph: Avatar Groups contains a list of companies, groups or users visually through profile images or initials
+tags: ["Profile picture", "Display picture"]
+githubLabels: ["component:avatar-group"]
+needToKnow:
+- An Avatar Group must have at least one Avatar provided
+- The `size` set in Avatar Group will supersede any passed in at the Avatar component level.
+- Use an avatar to help a user in the platform efficiently identify another person or an organization.
+- Use a profile avatar to represent an individual and an account avatar for an organization.
+- Company Avatars will not display initials and should be accompanied by an avatarSrc.
+- Has RTL support could have internationalisation support for the aria response in the render counter.
```

Comment:
Feels like this is 3 sentences in one ;) Is there bullet point missing in the middle somewhere?

Reply:
I've broken it out into two more succinct points 👍